scripts/lc-builds/blueos_xl-2018.11.26_omptarget.sh

```diff
@@ -26,7 +26,7 @@ cmake \
   -C ../host-configs/lc-builds/blueos/xl_2018_11_26.cmake \
   -DENABLE_OPENMP=On \
   -DENABLE_TARGET_OPENMP=On \
-  -DOpenMP_CXX_FLAGS="-qoffload;-qsmp=omp;-qnoeh;-qnoinline;-qalias=noansi" \
+  -DOpenMP_CXX_FLAGS="-std=c++11;-qoffload;-qsmp=omp;-qnoeh;-qnoinline;-qalias=noansi" \
```

Comment:
Is `-std=c++11` really required as an OpenMP flag?

Reply:
Good catch.  I'lll fix that when I get back to my office after another meeting.